test-data/unit/check-expressions.test

```diff
@@ -1157,6 +1157,15 @@ b'%c' % (123)
 [case testUnicodeInterpolation_python2]
 u'%s' % (u'abc',)
 
+
+-- F-String
+-- --------
+
+
+[case testFStringParseOk]
+a = f'foobar'
```

Comment:
Thanks for the tip, better than `python runtests.py testcheck`.

I've just added another test to check type checking of expressions but I can't get it working.
If I put the content in a `test_f_string.py` file and run `mypy --fast-parser --python-version 3.6 --show-traceback  test_f_strings.py` it runs without any error.

However running the test with `pytest -n0 -k testFStringParseOk` gives me:
```
___________________________________________________ testFStringTypecheckExpression ____________________________________________________
data: /Users/achauve/dev/github.com/achauve/mypy/test-data/unit/check-expressions.test:1169:
../../mypy/test/testcheck.py:113: in run_case
    self.run_case_once(testcase)
../../mypy/test/testcheck.py:190: in run_case_once
    assert_string_arrays_equal(output, a, msg.format(testcase.file, testcase.line))
../../mypy/test/helpers.py:85: in assert_string_arrays_equal
    raise AssertionFailure(msg)
E   mypy.myunit.AssertionFailure: Invalid type checker output (/Users/achauve/dev/github.com/achauve/mypy/test-data/unit/check-expressions.test, line 1169)
-------------------------------------------------------- Captured stderr call ---------------------------------------------------------
Expected:
Actual:
  main:3: error: Variable annotation syntax is only suppoted in Python 3.6, use type comment instead (diff)
```
It doesn't use python 3.6 syntax. How can I specify it in the tests?